integration/README.md

```diff
@@ -4,37 +4,22 @@ Integration directory will be the place to host various NVIDIA FLARE integration
 approval process. 
 
 ## Requirements
-Each research project should create a sub directory with following requirements
-
-* sub directory name must be in ASCII string and no longer than 35 characters long
+Each integration project should create a sub directory with following requirements
+* sub directory name must be in ASCII string, all in lower case, and no longer than 35 characters long
 * Each project should include
   * README.md -- document must include
     * objective 
     * background
     * description
     * nvflare version used
-    * data download and preparation
-    * steps to run the code
-    * expected results
-  * code needed to run nvflare job or Pull Request
+  * code needed to run code or Pull Request
```

Comment:
This line is confusing to me.
Should it just be code?

Pull request is to merge the changes back into NVFlare.